pkg/users/accounts-list.js

```diff
@@ -18,72 +18,277 @@
+                      onClick={() => delete_account_dialog(account)}>
+            {_("Delete account")}
+        </DropdownItem>,
+    );
+
+    const kebab = (
+        <Dropdown toggle={<KebabToggle onToggle={isOpen => setKebabOpen(isOpen)} />}
+                isPlain
+                isOpen={isKebabOpen}
+                position="right"
+                dropdownItems={actions} />
+    );
+    return kebab;
+};
+
+const getAccountRow = (account, current, groups) => {
+    const admins = ['sudo', 'root', 'wheel'];
+    const userGroups = [];
+
+    groups.forEach((group) => {
+        if (group.userlist.find(accountName => accountName === account.name)) {
+            const isAdmin = !!admins.find(adm => adm === group.name);
+            userGroups.push({ name: group.name, members: group.userlist.length, isAdmin: isAdmin });
+        }
+    });
+
+    userGroups.sort((a, b) => {
+        if (a.isAdmin)
+            return -1;
+        if (b.isAdmin)
+            return 1;
+        if (a.members === b.members)
+            return a.name.localeCompare(b.name);
+        else
+            return b.members - a.members;
+    });
 
-        cockpit.location.go([account.name]);
+    const userGroupLabels = userGroups.map(group => {
+        const color = group.isAdmin ? "gold" : "cyan";
+        return (
+            <Label key={group.name} variant="filled" color={color}>
+                {group.name}
+            </Label>
+        );
+    });
+
+    let loginText = "";
+    let loginSortKey = null;
+    if (account.loggedIn) {
+        loginText = _("Logged in");
+        loginSortKey = "logged in";
+    } else if (!account.lastLogin) {
+        loginText = _("Never logged in");
+        loginSortKey = "never";
+    } else {
+        loginSortKey = new Date(account.lastLogin);
+        loginText = timeformat.dateTime(loginSortKey);
     }
 
-    return (
-        <Card onClick={click} onKeyPress={click} isSelectable isCompact>
-            <CardBody className="cockpit-account">
-                <UserIcon className="cockpit-account-pic" />
-                <div className="cockpit-account-real-name">{account.gecos.split(',')[0]}</div>
-                <div className="cockpit-account-user-name">
+    const columns = [
+        {
+            title: (
+                <span>
                     <a href={"#/" + account.name}>{account.name}</a>
-                    {current && <Badge className="cockpit-account-badge">{_("Your account")}</Badge>}
-                </div>
-            </CardBody>
-        </Card>
-    );
-}
+                    {current && <Badge className="pf-u-ml-lg" id="current-account-badge">{_("Your account")}</Badge>}
+                </span>
+            ),
+            sortKey: account.name,
+            props: { width: 25, },
+        },
+        {
+            title: account.gecos.split(',')[0],
+            props: { width: 20, },
+        },
+        {
+            title: account.uid.toString(),
```

Comment:
The toString is not needed. It will get implicitly casted.

Reply:
Looks like it is. Otherwise I get `(intermediate value).localeCompare is not a function`